examples/resources/istio/service-entries.yaml

```diff
@@ -0,0 +1,240 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: serviceentries
+  namespace: lyczeq
+  labels:
+    app.kubernetes.io/name: serviceentries
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  dataSources: |
+    {
+      "podSelector": {
+        "resource": {
+          "kind": "Pod",
+          "version": "v1",
+        },
+        "filter": "$matchByLabelSelector($item, $root.spec.workloadSelector.labels)"
+      },
+    }
+  details: |
+    {
+      "header": [
+        {
+          "source": "spec.location",
+          "name": "spec.location",
+        },
+        {
+          "source": "spec.resolution",
+          "name": "spec.resolution",
+        },
+      ],
+      "resourceGraph": {
+        "colorVariant": 2,
+        "dataSources": [
+          {
+            "source": "podSelector"
+          }
+        ]
+      },
+      "body": [     
+          {
+            "name": "summary",
+            "widget": "Panel",
+            "source": "spec",
+            "visibility": "$boolean($exists(data.hosts) or $exists(data.addresses) or $exists(data.subjectAltNames))",
+            "children": [
+              { "name": "spec.hosts", "source": "$item.hosts", "widget": "JoinedArray", "visibility": "$exists(*)"},
+              { "name": "spec.addresses", "source": "$item.addresses", "widget": "JoinedArray", "visibility": "$exists(*)"},
+              { "name": "spec.exportTo", "source": "$item.exportTo", "widget": "Labels", "placeholder": "Exported to all Namespaces"},
+              { "name": "spec.subjectAltNames", "source": "$item.subjectAltNames", "widget": "JoinedArray", "visibility": "$exists(*)"},
```

Comment:
Previously we displayed here `Labels` and the title was configuration, if we want to follow our previous implementation it should be changed, if you have reason why it's like this it can stay, not strong opinion
![image](https://user-images.githubusercontent.com/46759954/190605179-b8d8f54c-f11b-474a-b43a-dcd9034fb318.png)

Reply:
I will change the title to the `Configuration` but I'd stay with the chosen widgets. We use normal strings for hosts in VirtualServices, so it would keep consistency. Same as addresses, in my opinion it's better to have them as strings not labels/tokens. I've chosen Labels widget for `exportTo` value because the elements can sometimes look like `*` or `.`, so it would be more readable if they are presented as labels/tokens.